app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -193,47 +214,108 @@ You will not be eligible for the IRP, because you have spent more than 90 days i
+
+#### Example 1
+You arrive in the UK in September 2024 on a Youth Mobility Scheme visa. You are employed as a teacher on a contract lasting 1 academic year, and successfully apply for the first instalment of the IRP. At the end of the academic year, your school offers to make your contract permanent and sponsor you for a Skilled Worker visa for 3 years. As your Youth Mobility Scheme visa is only valid for 2 years, and you are planning to work as a teacher in England for longer than this, you agree to switch your visa. You apply for the second instalment of the IRP in September 2025. You remain eligible because the Youth Mobility Scheme visa and the Skilled Worker visa are on the list of eligible visas on this page. 
+
+#### Example 2
+You arrive in the UK in September 2024 on a Youth Mobility Scheme visa. You are employed as a teacher on a contract lasting 2 academic years and successfully apply for the first instalment of the IRP. Early in 2025, your partner moves to the UK on a Skilled Worker visa lasting 3 years. You visit family overseas during the summer holidays. As your Youth Mobility Scheme visa is limited to 2 years, you decide to apply to join your partner as a dependant on their visa instead. You return to the UK to continue your employment in September 2025 on a dependant visa. You are not eligible for the second instalment of the IRP as the dependant visa does not appear on the list of eligible visas on this page.
 
-* employed by a recruitment agency – an agency can find and place you in a school, but your contract must be with the school and your salary must be paid directly to you by the school, not by the recruitment agency
 
-* teaching in Scotland, Wales or Northern Ireland
 
 ## How to apply for the international relocation payment (IRP)
 
-Applications for the international relocation payment (IRP) are now closed. You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We'll email you when applications reopen in autumn 2024.
+We will update this page with a link to the application form when it is live.
+
+Before you apply for the first and second instalments of the IRP, you’ll need to check you meet the eligibility requirements.
+
+Applications are open from 30 September 2024 to end of March 2025
+
+If you start your teaching job between 1 March 2024 and 28 February 2025 you should apply between 30 September 2024 and the end of March 2025. If you are eligible, you should receive the money following successful completion of our eligibility checks.You must apply between 30 September 2024 and the end of March 2025 to remain eligible.
+
+
+## DfE eligibility requirements for IRP instalments
+
+### First instalment of IRP
+
+DfE will check you meet the eligibility requirements for the first instalment of IRP by validating your application with:
+
+* [UK Visas and Immigration](https://www.gov.uk/government/organisations/uk-visas-and-immigration) 
```

Comment:
Is this link needed? One for policy

Reply:
Will check with Amy but agree, it's confusing to have both